src/languages/en.js

```diff
@@ -507,6 +507,8 @@ export default {
         chatYouLookingForCannotBeFound: 'The chat you are looking for cannot be found.',
         getMeOutOfHere: 'Get me out of here',
         iouReportNotFound: 'The payment details you are looking for cannot be found.',
+        notHere: "Hmm... it's not here",
```

Comment:
Would any of you be willing to link me to a convo where we decided on this copy?
Seems pretty uninformative to me.

Reply:
Ah, here: https://github.com/Expensify/App/issues/9522#issuecomment-1171702598
👍